### DISCUSSION TITLE
Get radar station from gridpoint or lat/lon?

### DISCUSSION THREAD

**Discussion by swbrains** [weather-gov/api#727] [Q&A] [answered]
Hi, I would like to be able to pull the radar loop gif from https://radar.weather.gov/ridge/standard/ for a particular area.  However, I need to get the radar station ID used in the names of the various loop gifs in this folder.  I already know the gridpoint and lat/lon coordinates for the location, but I don't know how to get the radar station ID that is used in the names of these gif files so I can make the connection and get the correct one for a particular area.   Is there an API endpoint that returns the radar station ID (or the full URL) for the radar images for that area?  Thanks!

**Comment by netbymatt** [comment, marked as answer]
By lookup up the point with latitude and longitude you also get a radar location. I have not seen this information in any other endpoint

Buffalo NY:  https://api.weather.gov/points/42.9414,-78.8928
```
{
  ...
  properties : {
    ...
     "radarStation": "KBUF"
  }
  ...
}
```

**Comment by swbrains** [comment]
Awesome -- Thanks!
